src/borrowedBooks.js

```diff
@@ -0,0 +1,40 @@
+const fs = require('fs'); //like import 
+const book = require('./book');
+
+//1. Read data from json file (contains array type)
+let rawdata = fs.readFileSync('./src/data/borrows.json');  
+let idObject = JSON.parse(rawdata);  //returns js arrays
+
+
+//returns list book ids 
+function getBookIds(idObject){
+    var bookIds=[]; 
+
+    for(var i = 0; i < idObject.length; i++){
+        let booksBorrowed = idObject[i].booksBorrowed; 
+        for(var j = 0; j < booksBorrowed.length; j++){
+            bookIds.push(booksBorrowed[j]);
+        }
+    }
+
+    if(idObject.length == 0){  
+        return {message: 'There is not borrowed books yet.'};
+    }
+    return bookIds;
+}
+var idArray = getBookIds(idObject);
```

Comment:
getBookIds should be called inside getBorrowedBookList mehtod since it has no use to call when the file loads.